File: server/src/main/kotlin/ru/itis/favein/controllers/HomeController.java
package ru.itis.favein.controllers;

import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.GetMapping;

@Controller
public class HomeController {
    @GetMapping
    public String home(Model model) {
        model.addAttribute("user", "Martis");
        return "home/index";
    }
}

File: server/src/main/kotlin/ru/itis/favein/controllers/HomeController.kt
package ru.itis.favein.controllers

import org.springframework.stereotype.Controller
import org.springframework.ui.Model
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping

@Controller
class HomeController {
    @GetMapping
    fun index(model: Model): String {
        return "home/index"
    }
}